jvm/j2v8/BUILD

```diff
@@ -17,6 +17,9 @@ j2v8_platform("macos")
 
 j2v8_platform("linux")
 
+# TODO: These should probably be AARs?
```

Comment:
there's no android specific resource here, right? I think i stripped all that out in order to fit the debugger in the jvm repo?

Reply:
The main problem here is they rely on transitive AARs, so if you try to pull the JAR into a Gradle project, it'll technically work, but will blow up when it tries to process the transitive deps. So, the idea is to make this an Android only artifact up front to make it clearer where this is meant to be used.

That being said, I'm not sure that's a great way to do it. From looking at a lot of the Android libs, a lot of them are JARs if they don't have any bundled Android resources.